source/brailleDisplayDrivers/brailliantB.py

```diff
@@ -104,26 +70,52 @@ class BrailleDisplayDriver(braille.BrailleDisplayDriver):
 
 	@classmethod
 	def check(cls):
-		try:
-			next(_getPorts())
-		except StopIteration:
-			# No possible ports found.
-			return False
-		return True
+		return (bdDetect.arePossibleDevicesForDriver(cls.name)
+			or next(cls.getManualPorts(), None) is not None)
+
+	@classmethod
+	def _getTryPorts(cls, port):
+		for match in super(BrailleDisplayDriver,cls)._getTryPorts(port):
+			if not match.type==bdDetect.KEY_CUSTOM:
+				yield match
+			else:
+				try:
+					rootKey = _winreg.OpenKey(_winreg.HKEY_LOCAL_MACHINE, r"SYSTEM\CurrentControlSet\Enum\USB\{usbId}".format(usbId=match.id))
+				except WindowsError:
+					continue
+				with rootKey:
+					for index in itertools.count():
+						try:
+							keyName = _winreg.EnumKey(rootKey, index)
+						except WindowsError:
+							break
+						try:
+							with _winreg.OpenKey(rootKey, os.path.join(keyName, "Device Parameters")) as paramsKey:
+								yield bdDetect.DeviceMatch(bdDetect.KEY_SERIAL, match.id,
+									_winreg.QueryValueEx(paramsKey, "PortName")[0],
+									match.deviceInfo
+								)
+						except WindowsError:
+							continue
+
+	@classmethod
+	def getManualPorts(cls):
+		return braille.getSerialPorts(filterFunc=lambda info: "bluetoothName" in info)

```

Comment:
Indeed, this method wasn't there initially. I added it because there might come newer Brailliant devices with another bt prefix in the future which communicate using the same protocol. IN that case, people can always manually connect until there's an NVDA update available.